### ISSUE TITLE
Loses objectMode part of logic. Don't work cork/uncork

### ISSUE THREAD

**Issue by scarry1992** [maxogden/websocket-stream#146]
When I create a stream with option objectMode: false, I lose cork/uncork part of logic. That's happens in duplexify in https://github.com/maxogden/websocket-stream/blob/master/stream.js#L76. When I remove that initial stream by duplexify and leave stream = proxy only, everything works well. Can you tell me why you are doing stream by duplexify and what benefits it gives. And maybe I can help resolve this issue?

**Comment by mcollina** [comment]
This should be fixed, right?

**Comment by scarry1992** [comment]
@mcollina, do you read last comment in pr in websocket-stream? With cork logic first writing packet in stream separated for two socket frames. I would like your help for resolve it. I attached screenshots with callstacks of two writing operations.

**Comment by mcollina** [comment]
Sorry I didn’t understood the problem or your comment. Can you rephrase it please, and maybe post a PR with a failing test?

**Comment by scarry1992** [comment]
Sorry, test was been written and he is passed. But testing pr with mqtt.js shows - every packet was been separated for two chunks. And I attached two screenshots for getting your help;)

**Comment by mcollina** [comment]
I have no clue given those screenshots. I’ll need an example that say “yes/no” to check things out. This way to complex to debug for the time I have available.:

**Comment by scarry1992** [comment]
Ok then, I will attach example at Monday, ok?

**Comment by mcollina** [comment]
sure!

Il giorno sab 6 apr 2019 alle 13:55 scarry1992 <notifications@github.com>
ha scritto:

**Comment by mcollina** [comment]
Can you paste the code of the server?

**Comment by scarry1992** [comment]
@mcollina, I can`t share code of my server, but test same with wss://iot.eclipse.org:443/ws

**Comment by scarry1992** [comment]
@mcollina, I mean using with mqtt.js
